deepchem/models/tensorgraph/models/symmetry_function_regression.py

```diff
@@ -313,16 +344,14 @@ def build_graph(self):
           n_hidden,
           self.atom_number_cases,
           activation='tanh',
-          in_layers=[previous_layer, self.atom_numbers])
+          in_layers=[previous_layer, self.featurized])
       Hiddens.append(Hidden)
       previous_layer = Hiddens[-1]
 
     costs = []
     self.labels_fd = []
     for task in range(self.n_tasks):
-      regression = Dense(
-          out_channels=1, activation_fn=None, in_layers=[Hiddens[-1]])
-      output = BPGather(self.max_atoms, in_layers=[regression, self.atom_flags])
+      output = BPGather2(in_layers=[Hiddens[-1], self.featurized])
```

Comment:
@miaecle  This was actually a bug we found, since you shouldn't be adding any dense layers outside of the AtomicDifferentiatedDense (it introduces dependence on the ordering of the atoms.) So we just do a simpler reduction instead.